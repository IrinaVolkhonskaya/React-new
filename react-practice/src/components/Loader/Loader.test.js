import React from "react";
import Loader from "./Loader.tsx";

import { shallow } from "enzyme";
import Enzyme from "enzyme";
import Adapter from "enzyme-adapter-react-16";

Enzyme.configure({ adapter: new Adapter() });

describe("<Logo />", () => {
  it("render correctly Loader component", () => {
    const LoaderComponent = shallow(<Loader />);
    expect(LoaderComponent).toMatchSnapshot();
  });

  it("check h1 container by default", () => {
    const LoaderComponent = shallow(<Loader />);
    expect(LoaderComponent.find("h1").text()).toEqual("Loading...");
  });
});
