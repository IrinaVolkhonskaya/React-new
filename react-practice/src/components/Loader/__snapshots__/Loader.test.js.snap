// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Logo /> render correctly Loader component 1`] = `ShallowWrapper {}`;
